### PR TITLE
Line listing cases & contacts : Make delete button sticky and first

### PR DESCRIPTION
**Pull request by sandorbanvitagroup**

### CODE REVIEW

**Review by danielrus — APPROVED** [review]

File: src/app/shared/widgets/line-listing-new-cases/line-listing-new-cases.component.scss [hunk 1]
```diff
@@ -46,6 +49,10 @@
 }
 
 .button-remove {
+  position: sticky;
+  background-color: $white;
```
Comment (adriandumavitagroup) [inline src/app/shared/widgets/line-listing-new-cases/line-listing-new-cases.component.scss:53]: are we not using map for $white?

File: src/app/shared/widgets/line-listing-new-contacts/line-listing-new-contacts.component.scss [hunk 2]
```diff
@@ -46,6 +49,10 @@
 }
 
 .button-remove {
+  position: sticky;
+  background-color: $white;
```
Comment (adriandumavitagroup) [inline src/app/shared/widgets/line-listing-new-contacts/line-listing-new-contacts.component.scss:53]: also white, map?

File: src/app/shared/widgets/line-listing-new-cases/line-listing-new-cases.component.scss (hunk 1 above)
Comment (sandorbanvitagroup) [reply src/app/shared/widgets/line-listing-new-cases/line-listing-new-cases.component.scss:53]: We have a variable defined for white in variables.scss and it is used in a lot of places like this. Do you think we should not use it?

File: src/app/shared/widgets/line-listing-new-contacts/line-listing-new-contacts.component.scss (hunk 2 above)
Comment (sandorbanvitagroup) [reply src/app/shared/widgets/line-listing-new-contacts/line-listing-new-contacts.component.scss:53]: Same here.

**Review by adriandumavitagroup — APPROVED** [review]

**Pull request merged**